infra/src/main/java/com/redhat/service/bridge/infra/transformations/TransformationEvaluatorFactory.java

```diff
@@ -0,0 +1,7 @@
+package com.redhat.service.bridge.infra.transformations;
+
+public interface TransformationEvaluatorFactory {
+    TransformationEvaluator build(String template);
```

Comment:
do we have some sort of documentation (readme, javadoc ...), how the template string should looks like?

Reply:
it is mentioned in https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/main/TRANSFORMATIONS.md